setup.py

```diff
@@ -14,7 +35,7 @@
     license="BSD (3-clause)",
     url="https://github.com/NSLS-II/bluesky",
```

Comment:
Should we have the URL updated?
```suggestion
    url="https://github.com/bluesky/bluesky",
```